Cargo.toml

```diff
@@ -57,6 +57,12 @@ default-features = false
 version = "2.2.1"
 default-features = false
 
+[dependencies.zeroize]
+version = "1.3"
+default-features = false
+features = ["zeroize_derive"]
```

Comment:
I'm almost certain that every type that currently uses `derive(Zeroize)` can instead use `impl zeroize::DefaultIsZeroes for T`. If so, we can drop this feature:
```suggestion
```